lib/tokenable.rb

```diff
@@ -1,12 +1,12 @@
 # frozen_string_literal: true
 
+require 'jwt'
```

Comment:
I think we can get rid of this, as we only use JWT in authable atm, and it's required there.